charts/merlin/templates/merlin-deployment.yaml

```diff
@@ -311,11 +311,10 @@ spec:
             echo "Running Swagger UI..."
             /usr/share/nginx/run.sh
       {{- end }}
-
       volumes:
       - name: config
-        configMap:
-          name: {{ template "merlin.fullname" .}}-environments
+        secret:
+        {{- include "merlin.getEnvironmentSecret" . | indent 10 }}
```

Comment:
My [above comment](https://github.com/gojek/merlin/pull/331/files#r1073284723) can avoid you using indent here.